reg_tests/chgres_cube/driver.dell.sh

```diff
@@ -22,14 +22,9 @@
 
 set -x
 
-module purge
-module load EnvVars/1.0.2
-module load ips/18.0.1.163
-module load impi/18.0.1
+source ../../sorc/machine-setup.sh > /dev/null 2>&1
+source ../../modulefiles/build.$target
 module load lsf/10.1
```

Comment:
This module load is feeling lonely. Can you pull that into the modulefile?

Reply:
Fixed.